src/ownership/lifetimes-function-calls.md

```diff
@@ -1,6 +1,13 @@
 # Lifetimes in Function Calls
 
-In addition to borrowing its arguments, a function can return a borrowed value:
+_Example problem_: 
```

Comment:
I decided just to use the itemized list, no header is not necessary. I am editing this on Github, so I cannot check quickly how a particular header level looks. Am I expected to run it locally to check the final look of the page?

Reply:
> I am editing this on Github, so I cannot check quickly how a particular header level looks. Am I expected to run it locally to check the final look of the page?

Nah, it's great to edit it here — I much appreciate getting all the content changes since that's where the hard work lies!

In this case, I was suggesting that you stick to "semantic" markup instead of using italic to simulate a heading (though I realize that I'm guilty of the same in the Android section where I mark filenames before code blocks with italic :see_no_evil:  My plan there would be to eventually invent a new include-syntax so that I don't have to repeat myself).